configuration_version.go

```diff
@@ -258,7 +262,17 @@ func (s *configurationVersions) Upload(ctx context.Context, uploadURL, path stri
 		return err
 	}
 
-	req, err := s.client.NewRequest("PUT", uploadURL, body)
+	return s.UploadTarGzip(ctx, uploadURL, body)
+}
+
+// UploadTarGzip is used to upload Terraform configuration files contained a tar gzip archive.
+// Any stream implementing io.Reader can be passed into this method. This method is also
+// particularly useful for tar streams created by non-default go-slug configurations.
+//
+// **Note**: This method does not validate the content being uploaded an2:00pmd is therefore the caller's
```

Comment:
```suggestion
// **Note**: This method does not validate the content being uploaded and is therefore the caller's
```

Some kind of weird cmd-V problem 😆

Reply:
Woah! :rofl: